### PR TITLE
HOTFIX: display synchronize new remote project url errors

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Updated to display error message if setting url manually for sync new remote project and the server returns an error (such as unauthorized, forbidden, not found, etc)

**To test:**

1) Create a project as user A and note the project ID (will use this is step 6)
2) Login as user B
3) From the `Projects` dropdown select `Synchronize Remote Project`
4) Select a project from the drop down
5) Click the checkbox for `Set URL Manually`
6) Enter in the url `http://IP_ADDRESS:PORT/api/projects/PROJECT_ID`
7) Click the `Create Synchronized Project` button
8) The URL input box should have an error underneath (access is denied)

accessdenied

To test not found error repeat everything and for PROJECT_ID in step 6 set the id to a project id that doesn't exist in the database. Click the `Create Synchronized Project` button and you should see a `Resource Not Found` error

resourcenotfound

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
~* [ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/repositories/remote/resttemplate/IridaOAuthErrorHandler.java [hunk 1]
```diff
@@ -30,11 +30,14 @@ public void handleError(ClientHttpResponse response) throws IOException {
 		logger.trace("Checking error type " + statusCode.toString());
 		switch (statusCode) {
 		case UNAUTHORIZED:
-			logger.trace("Throwing new IridaOAuthException for this error");
-			throw new IridaOAuthException("User is unauthorized for this service", remoteAPI);
+			logger.trace("Throwing new IridaOAuthException (Unauthorized) for this error");
+			throw new IridaOAuthException("User is unauthorized for this service", statusCode, remoteAPI);
 		case FORBIDDEN:
+			logger.trace("Throwing new IridaOAuthException (Forbidden) for this error");
+			throw new IridaOAuthException("User is forbidden from accessing this service", statusCode, remoteAPI);
```
Comment (ericenns) [inline src/main/java/ca/corefacility/bioinformatics/irida/repositories/remote/resttemplate/IridaOAuthErrorHandler.java:37]: Can you change `service` here to `resource`.

File: src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -3023,3 +3023,7 @@ AnalysisState.ERROR=Error
 # InputWithOptions                                                                        #
 #=========================================================================================#
 InputWithOptions.required={0} is required to run this pipeline. Please select one of the options.
+
+IridaOAuthErrorHandler.unauthorized=User is unauthorized for this service
+IridaOAuthErrorHandler.forbidden=Access denied. User is forbidden from accessing this service
```
Comment (ericenns) [inline src/main/resources/i18n/messages.properties:3028]: Can you update this error message to `Access denied. User is forbidden from accessing this resource`

**Review by ericenns — CHANGES_REQUESTED** [review]
This looks good, just a minor comment on the error message for FORBIDDEN.

File: src/main/java/ca/corefacility/bioinformatics/irida/repositories/remote/resttemplate/IridaOAuthErrorHandler.java (hunk 1 above)
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/repositories/remote/resttemplate/IridaOAuthErrorHandler.java]: Updated in 4d292bf1bc247ddd4230213ae0fed6957cf444a8

**Review by ericenns — APPROVED** [review]
👍 Thanks for fixing this bug!

**Pull request merged**